placeholders.go

```diff
@@ -46,6 +47,18 @@ func parsePlaceholders(input string, r *http.Request) string {
 			}
 			input = strings.Replace(input, "{user}", user, -1)
 		}
+		if strings.HasPrefix(placeholder[0], "{label") {
+			nStr := placeholder[0][6 : len(placeholder[0])-1] // get the integer N in "{labelN}"
+			n, err := strconv.Atoi(nStr)
+			if err != nil || n < 1 {
+				input = strings.Replace(input, placeholder[0], "", -1)
+			}
+			labels := strings.Split(r.Host, ".")
```

Comment:
Not really a concern here as we don't use this part of the host, but to be consistent I think we don't need the port and could just use `r.URL.Hostname()`